src/AutoRest.CSharp.V3/Output/Builders/ClientBuilder.cs

```diff
@@ -294,9 +294,9 @@ private static Paging GetClientMethodPaging(Method method, Method nextPageMethod
             BuilderHelpers.ParseConstant(requestParameter),
             requestParameter.Required == true);
 
-        private ResponseHeaderGroupType? BuildResponseHeaderModel(Operation operation, HttpResponse httpResponse)
+        private ResponseHeaderGroupType? BuildResponseHeaderModel(Operation operation, HttpResponse? httpResponse)
         {
-            if (!httpResponse.Headers.Any())
+            if (!(httpResponse?.Headers.Any() ?? false))
```

Comment:
Super hard to read.

Reply:
I've always followed this kind of logic: https://softwareengineering.stackexchange.com/a/314003/108631
I'm assuming you want me to write `if(!(httpResponse?.Headers.Any() == true))` instead, or separate it to a local variable. Unfortunately, C# doesn't handle tri-state booleans well syntactically. 😓

Reply:
I ended up writing it as `if (httpResponse == null || !httpResponse.Headers.Any())`, which is the most straightforward to read, and allows us to no longer need the `!` at the bottom of the method.